truckersmp-cli

```diff
@@ -788,6 +810,8 @@ if __name__ == "__main__":
         i = 0 if args.proton else 1
         compat_tool, start_game = start_functions[i]
         logging.debug("Starting game with {}".format(compat_tool))
+        if not check_libsdl2():
+            sys.exit("SDL2 is not found on your system.")
```

Comment:
```suggestion
            sys.exit("SDL2 wasn't found on your system.")
```

```suggestion
            sys.exit("SDL2 not found")
```

Reply:
Changed to "SDL2 was not found on your system."